app/uk/gov/hmrc/gform/views/form/snippets/group.scala.html

```diff
@@ -56,12 +56,10 @@
 
   @snippets
 
-  @formAction(prefix: String, mongoIdentifier: String) = @{ prefix + "-" + mongoIdentifier }
-
   @canAddAnother.fold(HtmlFormat.empty){ modelComponentId =>
     @govukButton(Button(content = Text(groupField.repeatAddAnotherText.map(ls => ls.value).getOrElse(messages("linkText.addRepeatingGroup"))),
         inputType=Some("submit"),
-        attributes = Map("formaction" -> uk.gov.hmrc.gform.gform.routes.FormController.updateFormData(formTemplateId, maybeAccessCode, sectionNumber, FastForward.Yes, AddGroup(modelComponentId)).path),
+        attributes = Map("id" -> "addRepeatingGroup", "formaction" -> uk.gov.hmrc.gform.gform.routes.FormController.updateFormData(formTemplateId, maybeAccessCode, sectionNumber, FastForward.Yes, AddGroup(modelComponentId)).path),
```

Comment:
Static **id** could cause accessibility issue like duplicate ids.

Reply:
It was initially a dynamic id, just like delete. However I think tests use a single group in pages, so its no a concern . FYI, thats how it was before as well.

Reply:
I think, you haven't pushed your local changes yet.

Reply:
Its already there, this is how Ratheesh wanted it. I have told him about your concerns, he will get back to me